src/trunk.c

```diff
@@ -4838,9 +4851,12 @@ trunk_compact_bundle(void *arg, void *scratch_buf)
    }
 
    platform_status pack_status = btree_pack(&pack_req);
-   platform_assert(SUCCESS(pack_status),
-                   "platform_status of btree_pack: %d\n",
-                   pack_status.r);
+   if (!SUCCESS(pack_status)) {
+      platform_default_log("btree_pack failed: %d\n", pack_status.r);
```

Comment:
For easier debugging, please consider changing this to print `"... failed: %d (%s)\n"` and print `platform_status_to_string()` for the `%s` arg.

Reply:
Nice!  Didn't know about that.